src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -401,12 +400,7 @@ export const gravityStitchingEnvironment = (
             if (!isExisty(description) || typeof description !== "string")
               return null
 
-            const { type: formatType } = Format
-            const desiredFormat = formatType
-              ?.getValues()
-              ?.find((e) => e.name === format)?.value
-
-            return formatMarkdownValue(description, desiredFormat)
```

Comment:
We could have updated this block to the following instead, but because the arg input is already typed... I'm not sure it adds much value to refetch from the enum:

```typescript
const desiredFormat = FormatEnums
  .getValues()
  .find((e) => e.value === format)?.value
```